test/src/unit-capi-string_dims.cc

```diff
@@ -854,6 +858,8 @@ int StringDimsFx::tiledb_array_get_non_empty_domain_var_size_from_name_wrapper(
           (tiledb_serialization_type_t)tiledb::sm::SerializationType::CAPNP,
           1) == TILEDB_OK);
 
+  tiledb_buffer_free(&buff);
```

Comment:
similar to previous comment, should probably init buffer = nullptr in its declaration.